src/pages/staticPages/tools/oncoprinter/OncoprinterGeneticUtils.ts

```diff
@@ -509,7 +551,12 @@ export function annotateGeneticTrackData(
                 }
                 if (d.molecularProfileAlterationType === AlterationTypeConstants.MUTATION_EXTENDED) {
                     // tag mutations as putative driver, and filter them
-                    d.putativeDriver = !!(d.oncoKbOncogenic || (params.useHotspots && d.isHotspot) || getCBioAnnotation(d));
+                    d.putativeDriver = !!(
+                        d.oncoKbOncogenic ||
```

Comment:
is this covered in test?  seems like a complicated condition that would qualify as a "unit" for test

Reply:
ehhhh... its realy just an OR statement, it looks a little ugly but its logically just saying that its a putative driver if any of the selected sources give it a driver annotation

Reply:
its implicitly covered by the e2e tests about annotation